assets/xml/objects/object_boss01.xml

```diff
@@ -1,189 +1,232 @@
+        <Animation Name="gOdolwaSpinSwordAnim" Offset="0x1AAF4" /> <!-- MM3D name is "dance_roll", but it was probably renamed. Might have originally been "jmp_stop03" from surrounding context. -->
+        <Animation Name="gOdolwaVerticalHopAnim" Offset="0x1AF18" /> <!-- MM3D name is "dance_jump", but it was probably renamed. Might have originally been "jmp_stop04" from surrounding context. -->
+        <Animation Name="gOdolwaHipShakeDanceAnim" Offset="0x1BA94" /> <!-- MM3D name is "dance_moth", but it was probably renamed. Might have originally been "jmp_stop05" from surrounding context. -->
+        <Animation Name="gOdolwaUpAndDownDanceAnim" Offset="0x1C700" /> <!-- MM3D name is "dance_fire", but it was probably renamed. Might have originally been "jmp_stop06" from surrounding context. -->
+        <Animation Name="gOdolwaArmSwingDanceAnim" Offset="0x1CE88" /> <!-- Original name is "jmp_stop07" -->
+        <Animation Name="gOdolwaThurstAttackAnim" Offset="0x1D8C8" /> <!-- Original name is "jmp_stop08" -->
+        <Animation Name="gOdolwaDoubleSlashAnim" Offset="0x1E68C" /> <!-- Original name is "jmp_stop09" -->
+        <Animation Name="gOdolwaSideToSideHopAnim" Offset="0x1EEA8" /> <!-- Original name is "jmp_stop10" -->
+        <Animation Name="gOdolwaSideToSideDanceAnim" Offset="0x1F6A4" /> <!-- Not present in MM3D, but might have been originally named "jmp_stop11" from surrounding context. -->
+        <Animation Name="gOdolwaSpinDanceAnim" Offset="0x1FC10" /> <!-- Original name is "jmp_stop12" -->
+        <Animation Name="gOdolwaJumpDanceAnim" Offset="0x204AC" /> <!-- Not present in MM3D, but might have been originally named "jmp_stop13" from surrounding context. -->
+
+        <!-- Unused vertices of unknown purpose. They seem similar to the vertices used for the bug's body. -->
+        <Array Name="gOdolwaUnusedVtx" Count="16" Offset="0x206F0">
+            <Vtx/>
+        </Array>
+
+        <!-- Odolwa Bug Limb DisplayLists -->
+        <DList Name="gOdolwaBugLightBodyDL" Offset="0x21230" />
+        <DList Name="gOdolwaBugDarkBodyDL" Offset="0x213A8" />
+        <DList Name="gOdolwaBugFrontLeftLegUpperDL" Offset="0x21518" />
+        <DList Name="gOdolwaBugFrontLeftLegLowerDL" Offset="0x21538" />
+        <DList Name="gOdolwaBugBackRightLegUpperDL" Offset="0x21570" />
+        <DList Name="gOdolwaBugBackRightLegLowerDL" Offset="0x21590" />
+        <DList Name="gOdolwaBugBackLeftLegUpperDL" Offset="0x215C0" />
+        <DList Name="gOdolwaBugBackLeftLegLowerDL" Offset="0x215E0" />
+        <DList Name="gOdolwaBugMiddleRightLegUpperDL" Offset="0x21608" />
+        <DList Name="gOdolwaBugMiddleRightLegLowerDL" Offset="0x21628" />
+        <DList Name="gOdolwaBugMiddleLeftLegUpperDL" Offset="0x21658" />
+        <DList Name="gOdolwaBugMiddleLeftLegLowerDL" Offset="0x21678" />
+        <DList Name="gOdolwaBugFrontRightLegUpperDL" Offset="0x216A8" />
+        <DList Name="gOdolwaBugFrontRightLegLowerDL" Offset="0x216C8" />
```

Comment:
Any reason why these are LegLower and LegUpper instead of UpperLeg and LowerLeg?